flink-table/flink-sql-parser/src/main/resources/org.apache.flink.sql.parser.utils/ParserResource.properties

```diff
@@ -17,6 +17,5 @@
 # See wrapper class org.apache.calcite.runtime.CalciteResource.
 #
 MultipleWatermarksUnsupported=Multiple WATERMARK statements is not supported yet.
-OverwriteIsOnlyAllowedForHive=OVERWRITE expression is only allowed for HIVE dialect.
 OverwriteIsOnlyUsedWithInsert=OVERWRITE expression is only used with INSERT statement.
 PartitionIsOnlyAllowedForHive=PARTITION expression is only allowed for HIVE dialect.
```

Comment:
It seems incorrect that "PARTITION expression is only allowed for HIVE dialect", because with this PR, users can insert into/overwrite a partition w/o the Hive dialect. So we should change it to something like "Creating partitioned table is only allowed for HIVE dialect"?